pyproject.toml

```diff
@@ -16,7 +16,13 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 
-dependencies = ["pydantic==1.8.2"]
+dependencies = [
+  "pydantic>=2.4.2", # dolma does not work with very old versions of pydantic
+  "dolma@git+https://github.com/allenai/dolma.git", # Install from git until a 0.9.2 package is released
```

Comment:
Have you tested that this works? I feel like pyproject.toml reject links in the dependencies due to security. If it works feel free to resolve this comment

Reply:
Yes, it works. And yes, it is not very safe. It's temporary until a new package has been published.